core/controllers/admin.py

```diff
@@ -631,3 +632,23 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        username = self.payload.get('username')
```

Comment:
I believe there are other ways to automatically get the current user ID, from which you can derive the username. It may even just be self.username. Check base.py.

You don't need to pass a payload in from the frontend.

Reply:
Done.